From b5442b8d8462f7567cee3f1b5e40d5140fafc106 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Emilio=20L=C3=B3pez?= <turl@tuxfamily.org>
Date: Tue, 5 Jun 2012 02:55:40 +0000
Subject: [PATCH 574/944] arm: disable DRM and DRM_MALI for sun4i

They break the build if you use a bare metal toolchain (they need
__linux__ defined), and besides they're not mandatory (you might
not want to run X11)
---
 arch/arm/Kconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 8d8cf4e..0f4dc51 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -723,8 +723,6 @@ config ARCH_SUN4I
 	select ARCH_HAS_CPUFREQ
 	select ARM_L1_CACHE_SHIFT_6
 	select GPU_MALI
-	select DRM
-	select DRM_MALI
 	help
 	  This enables support for Allwinner Technology Co., Ltd A10 SoC based systems
 
-- 
1.8.0

